src/Analyzers/CSharp/Tests/UseObjectInitializer/UseObjectInitializerTests.cs

```diff
@@ -20,17 +20,6 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseObjectInitializer;
 [Trait(Traits.Feature, Traits.Features.CodeActionsUseObjectInitializer)]
 public partial class UseObjectInitializerTests
 {
-    private static async Task TestInRegularAndScriptAsync(string testCode, string fixedCode, OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary)
```

Comment:
Why did this change produce so much churn in the object init tests?

Reply:
This analyzer was using an old strategy for code fading that was interfering with some new testing we needed for the feature change. I converted the analyzer to the new style as a separate commit at the beginning of this PR.